sass/site/primary/_posts-and-pages.scss

```diff
@@ -412,7 +412,7 @@
 		.entry-thumbnail {
 			float: left;
 			margin: 0;
-			width: 42.452830188679%;
```

Comment:
@adiraomj Can you bring me up to speed on this change? Our search result images need to be `360 x 238`. This change reduces them to `292 x 193`.